### DISCUSSION TITLE
Detecting LCD Digits

### DISCUSSION THREAD

**Discussion by SpatialJ** [jomjol/AI-on-the-edge-device#1064] [General]
Im trying to read values from my Wärmemengenzähler that has an LCD display.  Im planning to use external LEDs because of the glare but was wondering wether the include tflite-files should work with my LCD or if it would be necessary to train a new network?
Would you think it should work once i get rid of the glare and maybe optimize the focus of the camera? For now i used dig-class11_1411_s2_q
![image](https://user-images.githubusercontent.com/374475/192097907-23cb8b41-1eea-45fb-a1e2-63d8216fc025.png)

Best,
Johannes

**Comment by jomjol** [comment]
This will work, once you have a sharp image and adjusted the ROIs according to the wiki
